CHANGELOG.md

```diff
@@ -11,7 +11,10 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- We added the ability to add the current selection to a newly created group. [#11449](https://github.com/JabRef/jabref/issues/11449)
 - We made new groups automatically to focus upon creation. [#11449](https://github.com/JabRef/jabref/issues/11449)
+- We added the ability to right-click on selected entries and create a group using them. [#11449](https://github.com/JabRef/jabref/issues/11449)
```

Comment:
I think, you based your new branch on your branch before - and not on main. This is OK, we can manage.

Pleae remove this line, because that is already in main (rephrased).

```suggestion
```

Strange that GitHubs shows the cahnge in `StateManager`. Main should include it.

Reply:
Maybe merge `upstream/main`?

Reply:
Will do - I kept trying to do that but the desktop client was having some issues with it.